### PR TITLE
A user can view the selected event details.

### PR DESCRIPTION
**Pull request by grs83**

### CODE REVIEW

**Comment by grs83** [comment]
screen shot 2018-04-04 at 10 18 19 am

File: components/Item-Modal/index.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+                image
+                  ? image.large ? image.large.url : './img/logo.png'
+                  : './img/logo.png'
+              }
+              style={{ width: '300px', objectFit: 'cover' }}
+            />
+          </div>
+          <div className="description" style={{ width: '500px' }}>
+            <div className="ui header">
+              {`Starts at: ${timeConverter(
```
Comment (thebearingedge) [inline components/Item-Modal/index.js]: Extract message formatting function that takes `start_time` and `end_time` and returns the full string message.

File: components/Item-Modal/index.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                }}
+              >
+                <p
+                  dangerouslySetInnerHTML={createMarkup(description)}
+                  style={{
+                    fontSize: '12px'
+                  }}
+                />
+              </div>
+              {venue_url ? (
```
Comment (thebearingedge) [inline components/Item-Modal/index.js]: Check out `&&` short circuiting.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Cool, just some tiny changes.

**Review by thebearingedge — APPROVED** [review]
🚀

**Pull request merged**